service/src/main/java/uk/gov/hmcts/reform/fpl/controllers/CaseSubmissionController.java

```diff
@@ -131,15 +127,10 @@ public AboutToStartOrSubmitCallbackResponse handleMidEvent(@RequestBody Callback
     }
 
     @PostMapping("/about-to-submit")
-    public AboutToStartOrSubmitCallbackResponse handleAboutToSubmitEvent(
-        @RequestBody CallbackRequest callbackRequest) {
+    public AboutToStartOrSubmitCallbackResponse handleAboutToSubmitEvent(@RequestBody CallbackRequest callbackRequest) {
         CaseDetails caseDetails = callbackRequest.getCaseDetails();
 
-        byte[] pdf = documentGeneratorService.generateSubmittedFormPDF(caseDetails,
-            Pair.of("userFullName", userDetailsService.getUserName())
-        );
-
-        Document document = uploadDocumentService.uploadPDF(pdf, buildFileName(caseDetails));
+        Document document = caseSubmissionService.generateSubmittedFormPDF(caseDetails);
 
         ZonedDateTime zonedDateTime = ZonedDateTime.now(ZoneId.of("Europe/London"));
```

Comment:
Isn't this essentially the same as Time.now()?

Reply:
not essentially. Time.now uses `.toLocalDateTime()`. Also, this is an existing code. Any value changing this now as it currently works?

Reply:
Probably not, just remarking on things